service/models/operations/optimize.py

```diff
@@ -59,23 +61,26 @@ def gen_risk_bound(self):
             return -self.risk_bound
 
 
-def objfun(x, initial_guess, objective_function_option, relative_importance):
+def objfun(x, optimize_interventions: list[InterventionObjective]):
     """
     Calculate the weighted sum of objective functions based on the given parameters.
 
     Parameters:
     x (list): The current values of the variables.
-    initial_guess (list): The initial guess values of the variables.
-    objective_function_option (list): List of options specifying the type of objective function for each variable.
-    relative_importance (list): List of weights indicating the relative importance of each variable.
+    optimize_interventions: The interventions which are being optimized over
 
     Returns:
     float: The weighted sum of the objective functions.
     """
 
     # Initialize the total sum to zero
     total_sum = 0
-
+    # TODO: Will be cleaning this up in the next PR. I want minimal changes per PR for readability.
+    relative_importance = [i.relative_importance for i in optimize_interventions]
+    initial_guess = [i.initial_guess for i in optimize_interventions]
+    objective_function_option = [
+        i.objective_function_option for i in optimize_interventions
+    ]
     # Calculate the sum of all weights, fallback to 1 if the sum is 0
     sum_of_all_weights = np.sum(relative_importance) or 1
```

Comment:
Use `1.0` instead of `1` to avoid integer division, just in case.

Reply:
I wanted to keep this PR smaller for readability. The scope of this is to clean, and correctly use multiple intervention types
I will update the objective function in the follow up PR 
I will update this in my follow up branch good call